resources/integration_test/tests/functions.yml

```diff
@@ -59,7 +59,7 @@ tests:
     - [ "array", 0, null, null, null, null, 1, 0, 0, null, 0, null, "array", null, null, 0, null, null, null ]
     - [ "bson", 0, null, null, null, null, 1, 0, 0, null, 0, null, "bson", null, null, 0, null, null, null ]
     - [ "dbPointer", 0, null, null, null, null, 1, 0, 2, null, 0, null, "dbPointer", null, null, 0, null, null, null ]
-    - [ "decimal", 0, null, null, null, null, 1, 0, 2, 0, 0, 0, "decimal", null, null, 0, null, null, null ]
```

Comment:
This change has to do with the `// TODO SQL-1068: Change to SqlDataType::DECIMAL` that was in  the now deleted `bson_type_info.rs`.